docs/release-notes/rl-0.7.md

```diff
@@ -196,3 +196,7 @@ everyone.
 [Soliprem](https://github.com/Soliprem)
 
 - Add LSP and Treesitter support for R under `vim.languages.R`.
+
+[nezia1](https://github.com/nezia1)
+
+- Add [biome](https://github.com/biomejs/biome) support for Typescript, CSS and Svelte.
```

Comment:
Add instructions on how to switch, you may refer to options with `[](#opt-vim.language...)`